sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -69,8 +70,9 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      *
      * @return A CertificateOperationProperties object.
      */
-    CertificateOperationProperties Value() const override { return m_value; }
+    KeyVaultCertificateWithPolicy Value() const override { return m_value; }
 
+    CertificateOperationProperties Properties() const { return m_properties; }
```

Comment:
We can avoid this getter method by making Properties a public field

Reply:
i want it private, see previous comment